lib/ctblsymm.gd

```diff
@@ -180,8 +180,7 @@ DeclareGlobalFunction( "CharValueSymmetric" );
 ##  </Description>
 ##  </ManSection>
 ##
-DeclareGlobalVariable( "CharTableSymmetric",
-    "generic character table of symmetric groups" );
+DeclareGlobalName( "CharTableSymmetric" );
```

Comment:
Note that the second argument to `DeclareGlobalVariable` always was ignored, so it's safe to discard it here and elsewhere.